jr-stree/src/test/java/com/fasterxml/jackson/jr/stree/JacksonJrTreeTestBase.java

```diff
@@ -44,11 +44,19 @@ protected String a2q(String json) {
         return json.replace("'", "\"");
     }
 
+    protected JSON jsonWithTreeCodec(JSON config) {
+        return JSON.builder()
+                // 13-Feb-2020, tatu: There are 2 different ways actually..
+//            .treeCodec(new JacksonJrsTreeCodec())
+                .register(new JrSimpleTreeExtension(config))
```

Comment:
This is backwards incompatible change, breaking all existing users. I don't think that is doable.